fractal_server/app/api/v1/dataset.py

```diff
@@ -50,6 +51,9 @@ async def create_dataset(
     """
     Add new dataset to current project
     """
+    logging.debug(f"POST /project/{project_id}/dataset")
```

Comment:
This is not needed, since we already have logs about the URL of the endpoint. What we need to add is information about the request body.